saleor/dashboard/group/views.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import unicode_literals
+
+from django.shortcuts import redirect, get_object_or_404
+from django.contrib import messages
+from django.contrib.auth.models import Group
+from django.template.response import TemplateResponse
+from django.utils.translation import pgettext_lazy
+
+from .forms import GroupPermissionsForm
+from ..views import staff_member_required
+
+
+@staff_member_required
+def groups_list(request):
+    groups = Group.objects.all()
+
+    ctx = {'groups': groups}
+    return TemplateResponse(request, 'dashboard/group/list.html', ctx)
+
+
+@staff_member_required
+def group_create(request):
+    form = GroupPermissionsForm(request.POST or None)
+
+    if form.is_valid():
+        form.save()
+        return redirect('/dashboard/groups')
+
+    ctx = {'group': 'Add new group', 'form': form}
```

Comment:
Each string that we display in UI should be marked for translation (we usually use `pgettext_lazy`). But in this case I wouldn't expect `group` to be the string. In views that create new instances we usually create an empty object e.g. `group = Group()` at the beginning of the function, that could be also passed as `instance` to the form. Then in template you could check if `group.pk` is set - if yes then you can show it's name, otherwise render a string 'Add new group'.

Reply:
Done